common/src/main/scala/com/gu/zuora/model/response/Responses.scala

```diff
@@ -34,7 +34,10 @@ object ZuoraErrorResponse {
 case class ZuoraErrorResponse(success: Boolean, errors: List[ZuoraError])
     extends Throwable(errors.asJson.spaces2) with ZuoraResponse {
 
-  def asRetryException: RetryException = new RetryNone(message = this.asJson.noSpaces, cause = this)
+  def asRetryException: RetryException = errors match {
+    case List(ZuoraError("UNKNOWN_ERROR", _)) => new RetryUnlimited(this.asJson.noSpaces, cause = this)
```

Comment:
Although this is technically a list I've never seen it contain more than one item...

Reply:
Is this definitely the form that the error deserialises to?

Reply:
Yep I'm pretty sure it does - there's an example in the logs [here](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logEventViewer:group=/aws/lambda/support-CreateZuoraSubscriptionLambda-PROD;filter=UNKNOWN_ERROR;start=2018-01-11T18:17:00Z;end=2018-01-11T18:19:00Z).